tests/login.py

```diff
@@ -3,15 +3,18 @@
 
 from selenium import webdriver
 from parameterized import parameterized
+import selenium
 
 
 class Login(unittest.TestCase):
     def setUp(self) -> None:
         self.browser = webdriver.Chrome(executable_path='/Users/ellie/Selenium/Automation2020_Summer/browsers/chromedriver')
         self.browser.get('http://hrm-online.portnov.com/')
+        # new comment
```

Comment:
hey, this line looks good, but have you considered doing it that way.... blah blah